mkdocs.yml

```diff
@@ -73,11 +73,11 @@ plugins:
       date_from_meta:
         as_creation: date
   - git-revision-date-localized
+  - table-reader
```

Comment:
The table-reader issue is due to outdated mkdocs on your local machine. The current file is correct for latest version. You should remove this change.